### ISSUE TITLE
[Bug] selection search doesn't work twice in a row with the same selected text

### ISSUE THREAD

**Issue by XxJibril** [mozilla-mobile/fenix#15030]
## Steps to reproduce
select a word or a group of words, once that is done a group of options show up (Copy/Cut/Search) using the search button opens a new tab with the search results of the selected text, now close the tab and try researching the exact same selection of words

### Expected behavior
Obviously clicking the search button should once again open a new tab with the search results but it doesn't

### Actual behavior
The tab that should open once again doesn't open unless you use a different selection (one less/extra character)

### Device information
* Android device: Samsung Galaxy A7 (2018), Model: SM-A750F/DS, Android 9.0
* Fenix version: 80.1.3

**Comment by s-ankur** [comment]
Duplicate of https://github.com/mozilla-mobile/fenix/issues/13199

**Comment by XxJibril** [comment]
oh sorry i didnt notice that
